### PR TITLE
feature: remove git inner user

### PR DESCRIPTION
**Pull request by littlejiancc**
#### What type of this PR
/kind feature

#### What this PR does / why we need it:
fix: remove git inner user

#### Which issue(s) this PR fixes:
[gittar 删除内置 git_inner_user](https://erda.cloud/erda/dop/projects/387/issues/all?id=208148&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIl0sImFzc2lnbmVlSURzIjpbIjEwMDEyNjEiXX0%3D&issueTable__urlQuery=eyJwYWdlTm8iOjF9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=506&type=TASK)

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |     remove git inner user         |
| 🇨🇳 中文    |      gittar删除内置 git inner user        |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Effet** [comment]
Can you prove how other part (such as pipeline) still work normally after deleting inner user?

File: modules/gittar/auth/auth.go [hunk 1]
```diff
@@ -224,7 +222,6 @@ func doAuth(c *webcontext.Context, repo *models.Repo, repoName string) {
 					return
 				}
 				c.Set("repository", gitRepository)
-				//c.Set("lock", repoLock.Lock)
```
Comment (sfwn) [inline modules/gittar/auth/auth.go:227]: Not related to this pr.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/gittar/auth/auth.go (hunk 1 above)
Comment (littlejiancc) [reply modules/gittar/auth/auth.go:227]: done

**Review by sfwn — APPROVED** [review]

**Pull request merged**